pages/overview/welcome/index.page.tsx

```diff
@@ -51,7 +51,7 @@ export default function Welcome() {
     {
       title: 'Other Resources',
       summary:
-        'Deserunt et fugiat do adipisicing enim in nostrud Lorem anim ut amet enim.',
+        'Explore our different resources, books, articles, courses, videos, podcasts, and papers, we welcome you to enjoy them.',
```

Comment:
```suggestion
        'Deserunt et fugiat do adipisicing enim in nostrud Lorem anim ut amet enim.',
```

Please don't modify the resources description. That section is going to be removed from release 3.